### ISSUE TITLE
Scanning Line

### ISSUE THREAD

**Issue by BBBhatt** [mebjas/html5-qrcode#771]
How can we add Scanning Line moving from top to bottom and vice versa inside qrbox???

**Comment by ROBERT-MCDOWELL** [comment]
yes it's possible, I used to open a feature request long time ago but no following happened.
